### PR TITLE
Vinventory Dupe Fix

### PR DESCRIPTION
**Pull request by RoberioJr**
Resolves #. 

#### Changes proposed in this pull request: 
- Fix the bug that allows you to duplicate virtual items using a vehicle or a box with the help of another player. (Opening the virtual inventory at the same time)

- [x] I have tested my changes and corrected any errors found

### CODE REVIEW

File: Altis_Life.Altis/core/vehicle/fn_openInventory.sqf [hunk 1]
```diff
@@ -30,6 +30,13 @@ ctrlSetText[3504,format [(localize "STR_MISC_Weight")+ " %1/%2",_veh_data select
 [_vehicle] call life_fnc_vehInventory;
 life_trunk_vehicle = _vehicle;
 
+_vehicle spawn {
+    waitUntil {
+        if ((_this getVariable ["trunk_in_use_by",player]) != player) then {closeDialog 0; hint "Another player tried to open the inventory of that vehicle!"};
```
Comment (DomT602) [inline Altis_Life.Altis/core/vehicle/fn_openInventory.sqf]: Could we add the hint to the stringtable?

File: Altis_Life.Altis/core/vehicle/fn_openInventory.sqf [hunk 2]
```diff
@@ -30,6 +30,13 @@ ctrlSetText[3504,format [(localize "STR_MISC_Weight")+ " %1/%2",_veh_data select
...
+        (isNull (findDisplay 3500))
```
Comment (DomT602) [inline Altis_Life.Altis/core/vehicle/fn_openInventory.sqf]: Can remove the outer brackets

**Review by DomT602 — CHANGES_REQUESTED** [review]
Just 2 small changes :)

File: Altis_Life.Altis/core/vehicle/fn_openInventory.sqf [hunk 3]
```diff
@@ -30,6 +30,13 @@ ctrlSetText[3504,format [(localize "STR_MISC_Weight")+ " %1/%2",_veh_data select
 [_vehicle] call life_fnc_vehInventory;
 life_trunk_vehicle = _vehicle;
 
+_vehicle spawn {
+    waitUntil {
+        if ((_this getVariable ["trunk_in_use_by",player]) != player) then {closeDialog 0; hint localize "STR_MISC_VehInvUse"};
```
Comment (DomT602) [inline Altis_Life.Altis/core/vehicle/fn_openInventory.sqf]: Thanks for doing this, you do need to actually make the string in the stringtable though! If you need any help with that feel free to join the discord.

**Review by DomT602 — APPROVED** [review]
Looks good to me, just wanna check with someone else.

**Comment by RoberioJr** [comment]
Ok!

**Pull request closed without merging**